### ISSUE TITLE
Odd bug when selecting Help->About

### ISSUE THREAD

**Issue by csoghoian** [twstrike/coyim#224]
When I select Help->About, a semi-transparent window opens up, with no useful data within it. Once I click on the window, it becomes opaque, and then I can see the about information.

This is pretty obviously a bug. When this occurs, the following error appears in the terminal. 

```
(CoyIM:3965): Gtk-WARNING **: Failed to set text '<UNSET>' from markup due to error parsing markup: Error on line 1 char 24: Element 'markup' was closed, but the currently open element is 'UNSET'
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
```

Each time I go to Help->About the same error is displayed at the terminal, and the same transparent window is opened.

Screenshot attached. You can see the "About" window in the top left hand corner of the screen.

![coy-bug](https://cloud.githubusercontent.com/assets/4551147/12729569/79530948-c8f5-11e5-860d-65a496185d60.png)

**Comment by tcz001** [comment]
What's the gtk version you're using?

**Comment by csoghoian** [comment]
These are the gtk packages installed in the fedora qubes vm that i'm using for coyim.

gtk3-3.18.6-1.fc23.x86_64
gtk2-2.24.29-1.fc23.x86_64

**Comment by olabini** [comment]
I'm using the same Fedora packages, with no problems. Very strange. We should definitely set a parent for the GtkDialog, but that usually doesn't cause these kinds of problems. Hmm.

**Comment by juniorz** [comment]
c9fe1d9 sets the transient parent for the about dialog, but it had no effect.

**Comment by tcz001** [comment]
This can be css overwritten. Can you check gtk.css in the theme?

**Comment by juniorz** [comment]
I suspect it has been fixed by 958c0654 - [this line](https://github.com/twstrike/coyim/commit/958c0654#diff-a011a5252c872ee6d73f6541acc37f3aL428) to be specific.

**Comment by olabini** [comment]
Can we get a check whether this has been fixed or not, @csoghoian - thanks?
